docs/specs/tripgo.swagger.yaml

```diff
@@ -3262,8 +3262,17 @@ definitions:
               dismount:
                 type: boolean
                 description: Indicator for dismount sections in cycle paths (e.g., steps).
+              name:
+                type: string
+                description: Street name, for walking and cycling segments only.
+              cyclingNetwork:
+                type: string
+                description: Cycling network name, for cycling segments only. In case of a street belonging to more than one cycling route, this is the one that is travelled for the longest distance in the segment.
+              metres:
+                type: integer
+                description: Travel distance, for walking and cycling segments.
             required:
-              - encodedPolyline
+              - encodedWaypoints
```

Comment:
Isn't this a backwards-incompatible breaking change?

Reply:
@nighthawk, this is just a fix to the docs, since I noticed that our output actually says `encodedWaypoints`, not `encodedPolyline`. Sorry the confusion!

Reply:
Ah, thanks, all good then.